### PR TITLE
Lesson 1 4

### PR DESCRIPTION
**Pull request by tungphamwd**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+  const [todoList, setTodoList] = useState([]);
+  const addTodo = (newTodo) => {
+    setTodoList([...todoList, newTodo]);
+  };
   return (
     <>
-      <TodoList />
-      <AddTodoForm />
+      <TodoList todoList={todoList} />
+      <AddTodoForm onAppTodo={addTodo} />
```
Comment (riousenkai) [inline src/App.js:12]: let's move `AddTodoForm` on top of `TodoList`

**Review by riousenkai — APPROVED** [review]
Loosk good to me, but please address the comment I left on your next assignment

File: src/App.js (hunk 1 above)
Comment (tungphamwd) [reply src/App.js:12]: Thank you very much.

**Comment by tungphamwd** [comment]
Thank you for your reviewing @riousenkai .

**Pull request merged**